### PR TITLE
#1601: Add release notes for v2.17.0

### PR DESCRIPTION
**Pull request by benedeki**

### CODE REVIEW

File: _docs/2.0.0/usage/menas-api.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: docs
+title: Usage - Menas API
```
Comment (Zejnilovic) [inline _docs/2.0.0/usage/menas-api.md:3]: Why is this here when I have an open PR for it?

File: _posts/2020-11-13-release-2.17.0.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Release v2.17.0
+tags: ["v2.17.0", "changelog"]
```
Comment (Zejnilovic) [inline _posts/2020-11-13-release-2.17.0.md:4]: Did the Release Notes generator work?

File: _posts/2020-11-13-release-2.17.0.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<!--more-->
+
+
+
+## Menas Improvements and Fixes
+
+- [#763] ({{ site.github.issues_url }}/763) UI now displays projected after-conformance schema for datasets.
+- [#1518] ({{ site.github.issues_url }}/1518) Added export and import feature for Schema, Dataset and MappingTable. See the documentation of the REST API for [export]({{ site.baseurl }}/docs/usage/menas-api#export) and [import]({{ site.baseurl }}/docs/usage/menas-api#import).
+
+## Helper scripts Improvements
```
Comment (Zejnilovic) [inline _posts/2020-11-13-release-2.17.0.md:18]: missing https://github.com/AbsaOSS/enceladus/issues/1587

File: _docs/2.0.0/usage/menas-api.md (hunk 1 above)
Comment (benedeki) [reply _docs/2.0.0/usage/menas-api.md:3]: Was not aware of it. Will remove.

File: _posts/2020-11-13-release-2.17.0.md (hunk 2 above)
Comment (benedeki) [reply _posts/2020-11-13-release-2.17.0.md:4]: Nope, it didn't. Had to do it manually.

File: _posts/2020-11-13-release-2.17.0.md (hunk 3 above)
Comment (benedeki) [reply _posts/2020-11-13-release-2.17.0.md:18]: Thanks, will add.

File: _posts/2020-11-13-release-2.17.0.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+## Menas Improvements and Fixes
+
+- [#763] ({{ site.github.issues_url }}/763) UI now displays projected after-conformance schema for datasets.
+- [#1518] ({{ site.github.issues_url }}/1518) Added export and import feature for Schema, Dataset and MappingTable. See the documentation of the REST API for [export]({{ site.baseurl }}/docs/usage/menas-api#export) and [import]({{ site.baseurl }}/docs/usage/menas-api#import).
+
+## Helper scripts Improvements and Fixes
+
+- [#1546] ({{ site.github.issues_url }}/1546) Added version to _Helper scripts_ and this version is now printed with a banner at the beginning of any _Helper script_ run.
+- [#1592] ({{ site.github.issues_url }}/1592) _Helper scripts_ now have a setting to either print a warning or do a non-zero exit in case unknown parameters are passed to them. This config variable `EXIT_ON_UNRECOGNIZED_OPTIONS` can be found in `enceladus_env.sh.template`. Can be set to `"true"` or `"false"`.
+- [#1587] ({{ site.github.issues_url }}/1587) Value for `--null-value` option in helper scripts is properly encapsulated in `"` not to lose white spaces.
```
Comment (Zejnilovic) [inline _posts/2020-11-13-release-2.17.0.md:22]: This is not true. `--null-value " alfa "` will still lose the whitespaces. The fix handles empty spaces, not complex values. For complex values use `--null-value "\" alfa \""`

Comment (benedeki) [reply _posts/2020-11-13-release-2.17.0.md:22]: Hmm, I think I was testing it, and remember it kept the whitespaces... 🤔 Will double-check tomorrow.

Comment (benedeki) [reply _posts/2020-11-13-release-2.17.0.md:22]: You are right (of course, you wrote the feature 😉 ) Isn't it a problem actually?
Going to reword the sentence.

**Review by Zejnilovic — APPROVED** [review]
